### PR TITLE
system variables: add `tidb_log_file_max_days`

### PR DESCRIPTION
**Pull request by xhebox**
Signed-off-by: xhe <xw897002528@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs#6773
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@xhebox 这个变量是在 v5.3 中引入的吗？如果是，请勾选 v5.3 方框，并打上 v5.3 标签。谢谢~

**Comment by xhebox** [comment]
> @xhebox 这个变量是在 v5.3 中引入的吗？如果是，请勾选 v5.3 方框，并打上 v5.3 标签。谢谢~

确认了一下, 是在5.3上的.

File: system-variables.md [hunk 1]
```diff
@@ -927,6 +927,12 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`tikv,tiflash,tidb`
 - 这个变量用于设置 TiDB 在读取数据时可以使用的存储引擎列表。
 
+### `tidb_log_file_max_days` <span class="version-mark">从 v5.3 版本开始引入</span>
+
+- 作用域：SESSION
+- 默认值：`0`
+- 这个变量可以调整当前 TiDB 实例上日志的最大保留的天数，默认值是实例配置文件中指定的值。此变量只影响当前 TiDB 实例上的配置，重启后丢失，且配置文件不受影响。
```
Comment (TomShawn) [inline system-variables.md:934]: 【实例配置文件中指定的值】，是否能链接到 TiDB Configuration File 文件对应的配置项呢？
```suggestion
- 这个变量可以调整当前 TiDB 实例上日志的最大保留天数，默认值是实例配置文件中指定的值。此变量只影响当前 TiDB 实例上的配置，重启后丢失，且配置文件不受影响。
```

**Comment by morgo** [comment]
@db-will do you mind reviewing this PR too?

**Review by db-will — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -927,6 +927,12 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`tikv,tiflash,tidb`
 - 这个变量用于设置 TiDB 在读取数据时可以使用的存储引擎列表。
 
+### `tidb_log_file_max_days` <span class="version-mark">从 v5.3 版本开始引入</span>
+
+- 作用域：SESSION
+- 默认值：`0`
+- 这个变量可以调整当前 TiDB 实例上日志的最大保留天数，默认值是实例配置文件中指定的值。此变量只影响当前 TiDB 实例上的配置，重启后丢失，且配置文件不受影响。
```
Comment (TomShawn) [inline system-variables.md:934]:
```suggestion
- 这个变量可以调整当前 TiDB 实例上日志的最大保留天数。默认值是实例配置文件中指定的值，见 [`max-days`](/tidb-configuration-file.md#max-days)。此变量只影响当前 TiDB 实例上的配置，重启后丢失，且配置文件不受影响。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**